lib/pure/bitopts.nim

```diff
@@ -0,0 +1,283 @@
+#
+#
+#            Nim's Runtime Library
+#        (c) Copyright 2017 Nim Authors
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+## This module implements a series of low level methods for bit manipulation.
+## By default, this module use fairly optimized procs written in pure nim.
+##
+## To improve performance, you can use the `builtinBitOpts` flag. This unlock
```

Comment:
I think this should be the default. And I'm afraid the implementations should use internally the ``when nimvm`` feature to provide the pure Nim implementation for compiletime evaluation purposes.